### PR TITLE
Add text anchor option

### PR DESCRIPTION
**Pull request by OlivierKamers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,7 +143,8 @@ font              | Arial     | Font-family of the text
     width: 1,
     fill: "#000000",
     color: "#ffffff",
-    font: "Calibri"
+    font: "Calibri",
+    anchor: "middle"
```
Comment (rugheid) [inline README.md]: Minor: comma at end?

File: src/staticmaps.js [hunk 2]
```diff
@@ -280,7 +280,8 @@ class StaticMaps {
                 font-size="${text.size}pt"
                 stroke="${text.color}"
                 fill="${text.fill ? text.fill : 'none'}"
-                stroke-width="${text.width}">
+                stroke-width="${text.width}"
+                text-anchor="${text.anchor}">
```
Comment (rugheid) [inline src/staticmaps.js]: Minor: > on next line?

**Review by rugheid — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (OlivierKamers) [reply README.md]: I've kept it consistent with the other examples in the readme

File: src/staticmaps.js (hunk 2 above)
Comment (OlivierKamers) [reply src/staticmaps.js]: Done

**Pull request merged**